### ISSUE TITLE
Disabling tonemapping

### ISSUE THREAD

**Issue by paulmelis** [Moguri/panda3d-simplepbr#46]
I'm trying to figure out why some colors in my rendering aren't what I expect them to be. To be sure the tone-mapping isn't involved I would like to disable it completely, but I don't see a flag for that. I thought setting exposure to 1.0 would disable it, but looking at this line it seems there is more going on:

https://github.com/Moguri/panda3d-simplepbr/blob/449335fa9381df68adad7128f6433e22d04a4e58/simplepbr/shaders/tonemap.frag#L30

**Comment by rdb** [comment]
It makes sense to offer a choice of different tone mappers, but I think disabling tone mapping makes no sense because it's fundamentally necessary to map the unbounded physically based luminance values to limited-range values that can be displayed on the screen.

**Comment by Moguri** [comment]
I agree that "disabling tone mapping" does not really make sense since the tone mapping is required to make the render output usable for most (any?) display.

I am going to close this, but I am still open to offering more choice and configuration in tone mappers.
